.github/workflows/slack.yml

```diff
@@ -29,28 +29,38 @@ jobs:
       - name: Install dependencies
         run: |
           python3 -m pip install --upgrade pip
-          pip3 install python-dotenv
+          # pip3 install python-dotenv
             
       # Runs a script using the runners shell
       - name: Run a user extraction script
         working-directory: ./metrics/slack          
         env:
           SLACK_KEY: ${{ secrets.SLACK_KEY }}
-          
+          MINIO: ${{ secrets.MINIO }}
```

Comment:
@venkuppu-chn  Not sure if we should use the product name in our repository. We have cleaned-up such words as part of contentious word removal activity. 
@johnbent  Could you please suggest.

Reply:
Manually signed-off the DCO this time. I will ensure the same from next time in commit message.